src/components/attendee/VideoComms/VideoCommsParticipant.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { ButtonConfig } from "../VideoHuddle/useVideoHuddle";
+
+import { AudioTrackPlayer } from "./internal/AudioTrackPlayer";
+import { ExtraButton } from "./internal/ExtraButton";
+import { VideoCommsControls } from "./internal/VideoCommsControls";
+import { Participant, VideoSource } from "./types";
+import { VideoTrackDisplay } from "./VideoTrackDisplay";
+
+interface VideoCommsParticipantProps {
+  participant: Participant;
+  isLocal?: boolean;
+  extraButtons?: ButtonConfig[];
+}
+
+export const VideoCommsParticipant: React.FC<VideoCommsParticipantProps> = ({
+  participant,
+  isLocal,
+  extraButtons = [],
```

Comment:
These might be better passed as `children`. Could also allow for other kinds of components.